readme/apps/mfa.md

```diff
@@ -0,0 +1,59 @@
+# Multi-factor authentication
+
+You can now enable Multi-Factor Authentication (MFA) to your Joplin Account.
+MFA enhances your account security by requiring more than just a password to log in.
+Joplin Cloud uses a method that generates a new code every 30s, this code needs 
+to be entered on each new log in after MFA is enabled.
+
+If you can't generate your time-based code because you have lost access to your 
+smartphone, for example, you will need to use one of the recovery codes to log in.
+
+## Recovery Codes
+
+When you enable MFA, Joplin Cloud will show you a list of codes,that you should save in 
+somewhere safe. Each code can only be used once, but you can generate new codes,
+if necessary, from your profile page.
+
+### How to enable MFA
+
+To enable MFA, you should have an app that will be able to store your MFA secret.
+There are many alternatives, and any app that is also a password manager should do it.
+
+Access your profile by clicking your full name in the top bar. Find the 
+*Multi-factor authentication* section and there should be a link to *Enable Multi-Factor Authentication*.
+
+On the next page, you will see a QRCode. You should now open your app preferred 
+authentication app in your phone and read the QRCode. At this point, you will 
+probably be prompted to store this as a Joplin Cloud credential. After the 
+process is finished you will see that every 30s a new code is generated.
+
+To finish enabling MFA, you should now enter two consecutive codes generated by
+the app in the input field presented and click *Enable MFA*.
+
+If successful you should see a notification and a new page with your recovery 
+codes, **remember to save them**.
+
+### How to disable MFA
+
+Access your profile by clicking your full name in the top bar. Find the 
+*Multi-factor authentication* section, and there should be link to *I want to disable it*.
+
+### I forgot to copy my recovery codes, how do I find them?
+
+If you have enabled MFA, but forgot to save your recovery codes, you can retrieve 
+them again by going to your profile and clicking the *Recovery Codes* link in the
+*Multi-factor authentication* section.
+
+Before viewing your recovery codes, you will be prompted to enter an authentication code
+(generated by your app) or your password. Every time you access your recovery code, 
+Joplin Cloud will send you an alert email.
+
+In this page, you will find a button at the bottom that will generate new codes,
+don't forget to copy the new codes if you opt to generate new ones.
+
+### How to log in with my recovery code
+
+If you need to make login with your recovery code, when you are prompt to enter
+your authentication code in the login screen you will see a link called
+*Use a recovery code*. In this screen you have a new input field for the recovery
+code.
```

Comment:
This doc is well written but I wonder if we should aim to make it shorter. Partly because, well, users don't like to read the manual, and partly because it's more to maintain and translate. Have a look at at how it's done for example on Dropbox or OneDrive: https://help.dropbox.com/account-access/enable-two-step-verification https://support.microsoft.com/en-us/account-billing/how-to-use-two-step-verification-with-your-microsoft-account-c7910146-672f-01e9-50a0-93b4585e7eb4

It's essentially click here, then follow the instructions. That means our instructions during the process should be clear enough, so that we can keep the doc here to a minimum.

Also I think any process we describe (click here, click there), should be in bullet points.

I think the section titles you have are fine, just the content should be more concise, or bullet points